pt_miniscreen/core/components/list.py

```diff
@@ -53,12 +53,31 @@ def __init__(
 
         self._rows_snapshot = None
         self._cleanup_transition = threading.Event()
+        self.visible_scrollbar = True
 
         # setup initial rows
         start_index = self.state["top_row_index"]
         end_index = self.state["num_visible_rows"] + self.state["top_row_index"]
         self.rows = [self.create_child(Row) for Row in Rows[start_index:end_index]]
 
+    @property
+    def visible_scrollbar(self):
+        return self._visible
+
+    @visible_scrollbar.setter
+    def visible_scrollbar(self, value):
+        self._visible = value
+
+    @property
+    def can_scroll_down(self):
```

Comment:
I like these as properties by the way, seems like a really common thing to want to know!